### PR TITLE
Remove nginx-controller

### PR DESCRIPTION
**Pull request by majakurcius**
**Description**

Changes proposed in this pull request:
- [x] Remove the following charts: 
   * Application Connector Ingress 
   * Application Connector Helper 
- [x] Remove `application-connector-ingress` and `application-connector-helper` from the Installation CR
- [x] Remove obsolete Application Connector overrides
- [x] Update [documentation](https://kyma-project.io/docs/components/application-connector/#tutorials-rotate-the-root-certificate-and-the-key-issued-by-the-certificate-authority-rotating-a-soon-to-expire-ca-certificate)

**Related issue(s)**
#5057

### CODE REVIEW

File: docs/application-connector/08-10-rotate-root-ca.md [hunk 1]
```diff
@@ -68,21 +68,25 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
...
   
    ```bash
    kubectl -n istio-system edit secret kyma-gateway-certs-cacert
    ```
 
-1. Renew the certificates in a Runtime. To do that, create a CertificateRequest custom resource (CR) in the Runtime in which you want to renew the certificates. Alternatively, wait for the certificates to expire in a given Runtime. The system renews the certificates automatically using the information stored in the Secrets you updated.
+1. Wait for all the client certificates to be renewed. 
 
-1. After the certificates are renewed in a Runtime, remove the `kyma-gateway-certs-cacert` Secret entry which contains the old certificate. First, encode the `new-ca.crt` file with base64.
+    > **NOTE:** In the case of a Kyma Runtime connected to the central Connector Service, the system renews the certificates automatically using the information stored in the Secrets. Alternatively, you can renew the certificates in said Runtime yourself. To do that, create a CertificateRequest custom resource (CR) in the Runtime in which you want to renew the certificates.
```
Comment (klaudiagrz) [inline docs/application-connector/08-10-rotate-root-ca.md]:
```suggestion
    > **NOTE:** In the case of a Kyma Runtime connected to the central Connector Service, the system renews the certificates automatically using the information stored in Secrets. Alternatively, you can renew the certificates in said Runtime yourself. To do that, create a CertificateRequest custom resource (CR) in the Runtime in which you want to renew the certificates.
```

File: docs/application-connector/08-10-rotate-root-ca.md [hunk 2]
```diff
@@ -68,21 +68,25 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
...
+
+1. After the client certificates are renewed, remove the `kyma-gateway-certs-cacert` Secret entry which contains the old certificate. First, encode the `new-ca.crt` file with base64.
   
+   > **CAUTION:** Do not proceed with this step until all the client certificates have been renewed!
```
Comment (klaudiagrz) [inline docs/application-connector/08-10-rotate-root-ca.md]:
```suggestion
   > **CAUTION:** Do not proceed with this step until all client certificates are renewed!
```

File: docs/application-connector/08-10-rotate-root-ca.md [hunk 3]
```diff
@@ -68,21 +68,25 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
...
+
    ```bash
    cat new-ca.crt | base64
    ```
 
-1. Edit the Secret and replace the `ca.crt` value with the `new-ca.crt` base64-encoded certificate.
+1. Edit the Secret and replace the `cacert` value with the `new-ca.crt` base64-encoded certificate.
```
Comment (klaudiagrz) [inline docs/application-connector/08-10-rotate-root-ca.md]:
```suggestion
1. Edit the Secret and replace the `cacert` value with the base64-encoded `new-ca.crt` certificate.
```

File: docs/application-connector/08-10-rotate-root-ca.md (hunk 1 above)
Comment (majakurcius) [reply docs/application-connector/08-10-rotate-root-ca.md]: It's on purpose here :)

File: docs/application-connector/08-10-rotate-root-ca.md (hunk 2 above)
Comment (majakurcius) [reply docs/application-connector/08-10-rotate-root-ca.md]: It's here for emphasis 'cause it's really important.

File: docs/application-connector/08-10-rotate-root-ca.md [hunk 4]
```diff
@@ -68,21 +68,25 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
...
+1. After the client certificates are renewed, remove the `kyma-gateway-certs-cacert` Secret entry which contains the old certificate. First, encode the `new-ca.crt` file with base64.
   
+   > **CAUTION:** Do not proceed with this step until all the client certificates are renewed!
+
    ```bash
    cat new-ca.crt | base64
    ```
 
-1. Edit the Secret and replace the `ca.crt` value with the `new-ca.crt` base64-encoded certificate.
+1. Edit the Secret and replace the `cacert` value with the base64-encoded `new-ca.crt` certificate.
```
Comment (franpog859) [inline docs/application-connector/08-10-rotate-root-ca.md]:
```suggestion
1. Edit the Secret and replace the `ca.crt` value with the base64-encoded `new-ca.crt` certificate.
```

File: docs/application-connector/08-10-rotate-root-ca.md [hunk 5]
```diff
@@ -68,21 +68,25 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
    cat merged-ca.crt | base64
    ```
 
-1. Replace the old certificate in the Istio Secret. Edit the Secret and replace the `ca.crt` value with the `merged-ca.crt` base64-encoded certificate.
+1. Replace the old certificate in the Istio Secret. Edit the Secret and replace the `cacert` value with the `merged-ca.crt` base64-encoded certificate.
```
[the remark this answers is not in the document]
Comment (majakurcius) [reply docs/application-connector/08-10-rotate-root-ca.md]: I thought the same and asked @Szymongib about it and he said it's actually `cacert` and that it should be renamed.

File: docs/application-connector/08-10-rotate-root-ca.md (hunk 4 above)
Comment (majakurcius) [reply docs/application-connector/08-10-rotate-root-ca.md]: As mentioned previously, I asked @Szymongib about it and he said it's actually cacert and that it should be renamed.

**Pull request merged**